tests/modules/programs/btop/example-settings.nix

```diff
@@ -0,0 +1,38 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.btop = {
+      enable = true;
+      package = config.lib.test.mkStubPackage { };
```

Comment:
This references a not-yet committed package option I suppose.

Reply:
done :)